melodic/distribution.yaml

```diff
@@ -3804,7 +3804,7 @@ repositories:
       url: https://github.com/ros-geographic-info/geographic_info.git
       version: master
     status: maintained
-  geomerty2_python3:
+  geometry2_python3:
```

Comment:
I think this now isn't in alpha order, that's why see is failing.
BTW, I think it's better to use `jsk_geometry2_python3`, so it's namespaced there as well.

Reply:
@ivanpauno, thank you, should be fixed